src/base/rss/rssfeed.cpp

```diff
@@ -399,6 +399,21 @@ void Feed::downloadArticleTorrentIfMatching(const ArticlePtr &article)
     BitTorrent::Session::instance()->addTorrent(torrentUrl, params);
 }
 
+QString Feed::generateSavePath(const QString &savePathTpl, const ArticlePtr &article)
+{
+    QRegExp rx("<(.+)>");
+    rx.setMinimal(true);
+    QString result = savePathTpl;
+    int pos = 0;
+    while ((pos = rx.indexIn(result, pos)) != -1) {
+        QString value = article->getValue(rx.cap(1));
+        result.replace(rx.cap(0), value);
```

Comment:
I am in no way a regexp expert, and my knowledge comes from JavaScript. So if there is someone who knows his way around this in QT, I'd be happy to see a better way of doing this. One thing comes to mind - the rx can be static.
From what I understand, the cap(0) is the complete capture, so if the string was "/downloads/<name>/some_other_folder", cap(0) will be "<name>", while cap(1) will be "name".
so we look for the vale of "name" in the rss article, and then replace "<name>" with the result.